packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -507,39 +539,12 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
         });
 
         describe('Preconfigured Tiers', () => {
-          const createAliasForNonBasicPlans = async (subscriptionTier: SubscriptionTier) => {
-            const aliasAccount = await Utils.createAliasAccount(
-              mirrorNode,
-              global.accounts[0],
-              requestId,
-              initialBalance,
-            );
-
-            const hbarSpendingPlan = await hbarSpendingPlanRepository.create(subscriptionTier, requestDetails, mockTTL);
-
-            await ethAddressSpendingPlanRepository.save(
-              { ethAddress: aliasAccount.address, planId: hbarSpendingPlan.id },
-              requestDetails,
-              mockTTL,
-            );
-
-            const plan = await ethAddressSpendingPlanRepository.findByAddress(aliasAccount.address, requestDetails);
-            expect(plan.ethAddress).to.eq(aliasAccount.address);
-            expect(plan.planId).to.eq(hbarSpendingPlan.id);
-            const spendingPlan = await hbarSpendingPlanRepository.findByIdWithDetails(plan.planId, requestDetails);
-            expect(spendingPlan.active).to.be.true;
-            expect(spendingPlan.amountSpent).to.eq(0);
-            expect(spendingPlan.subscriptionTier).to.eq(subscriptionTier);
-
-            return { aliasAccount, hbarSpendingPlan };
-          };
```

Comment:
**Note for reviewer:** This function was moved to a more general location and renamed `createAliasAndAssociateSpendingPlan()` to enable reuse in other tests.